python/requirements.txt

```diff
@@ -56,7 +56,7 @@ mypy
 networkx
 numba
 openpyxl
-pettingzoo>=1.3.2
+pettingzoo>=1.4.0
```

Comment:
To avoid always breaking travis when pettingzoo updates, should we pin this version in the future?

Reply:
Having a pinned version is fine as long as it is purely internal (which I believe it is here). Feel free to change this line to ==